edgelet/build/linux/install.sh

```diff
@@ -69,26 +69,26 @@ process_args "$@"
 
 install_rust
 
-# Add trusty repo to get older version of libc6-armhf-cross
-sudo add-apt-repository "deb http://archive.ubuntu.com/ubuntu/ trusty main universe"
-
 # Install OpenSSL, curl and uuid and valgrind
 sudo apt-get update || :
 sudo apt-get install -y \
     pkg-config \
```

Comment:
Per https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md, it looks like the `pkg-config` is preinstalled in 18.04

Reply:
I've been going back and forth on this. `pkg-config` is (apparently) a prerequisite for the logic in this script, and if 18.04 already has it then it will just be a no-op. So leaving it in doesn't impact the script's correctness.

I really want to keep this PR focused on upgrading agents. If we want to make a pass through the install script to streamline it or remove cruft, that can be done separately.